gallery/src/pages/components/ha-selector.ts

```diff
@@ -542,9 +542,6 @@ class DemoHaSelector extends LitElement implements ProvideHassElement {
   }
 
   static styles = css`
-    ha-settings-row {
-      --paper-item-body-two-line-min-height: 0;
```

Comment:
Won't this impact the functionality?

Reply:
This does make the ha-settings-row in height longer.